rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "stable"
+components = ["rustfmt", "clippy"]
```

Comment:
add rust-analyzer to components

Reply:
why ?

Reply:
so that when you run cargo install you get it installed with the correct version, without having to fill it manualy